libr/core/cmd_anal.c

```diff
@@ -1257,6 +1257,8 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
 	RAnalOp op;
 	ut64 addr;
 	bool isFirst = true;
+	int addrsize = r_config_get_i (core->config, "asm.bits");
```

Comment:
imho we should have a different variable for this and not depend on asm.bits, because that will break on thumb for example